lib/src/test/java/com/dynatrace/metric/util/DynatraceMetadataEnricherTest.java

```diff
@@ -72,96 +86,61 @@ public void testGetIndirectionFileContentValid() throws IOException {
-
-    StringReader reader = new StringReader(String.join("\n", expected));
-    List<String> result = DynatraceMetadataEnricher.getDynatraceMetadataFileContents(reader);
-    assertEquals(expected, result);
-    assertNotSame(expected, result);
-  }
-
-  @Test
-  public void testGetDynatraceMetadataFileContentInvalid() throws IOException {
-    List<String> inputs = Arrays.asList("=0", "", "a=", "\t\t", "=====", "    ", "   test   ");
-    List<String> expected = Arrays.asList("=0", "", "a=", "", "=====", "", "test");
-
-    StringReader reader = new StringReader(String.join("\n", inputs));
-    List<String> result = DynatraceMetadataEnricher.getDynatraceMetadataFileContents(reader);
-    assertEquals(expected, result);
-    assertNotSame(expected, result);
-  }
-
-  @Test
-  public void testGetDynatraceMetadataFileContentEmptyFile() throws IOException {
-    List<String> expected = new ArrayList<>();
-
-    List<String> result =
-        DynatraceMetadataEnricher.getDynatraceMetadataFileContents(new StringReader(""));
-    assertEquals(expected, result);
-    assertNotSame(expected, result);
-  }
-
-  @Test
-  public void testGetDynatraceMetadataFileContentPassNull() {
-    assertThrows(
-        IOException.class, () -> DynatraceMetadataEnricher.getDynatraceMetadataFileContents(null));
-  }
-
-  @Test
-  public void testGetMetadataFileContentWithRedirection_Valid() {
+  void testGetPropertiesWithIndirection_Valid() {
     // this should not be used. If it is, it will not exist and throw an exception, breaking the
     // test.
     String nonExistentAlternativeFilename = generateNonExistentFilename();
-    List<String> expected = Arrays.asList("key1=value1", "key2=value2", "key3=value3");
-    List<String> results =
-        DynatraceMetadataEnricher.getMetadataFileContentWithRedirection(
+    Properties expected = new Properties();
+    expected.setProperty("key1", "value1");
+    expected.setProperty("key2", "value2");
+    expected.setProperty("key3", "value3");
+
+    Properties results =
+        DynatraceMetadataEnricher.getPropertiesWithIndirection(
             "src/test/resources/indirection.properties", nonExistentAlternativeFilename);
     assertEquals(expected, results);
     assertNotSame(expected, results);
   }
 
   @Test
-  public void
-      testGetMetadataFileContentWithRedirection_IndirectionFileDoesNotExistAlternativeDoesNotExist() {
+  void testGetPropertiesWithIndirection_IndirectionFileDoesNotExistAlternativeDoesNotExist() {
     String filename = generateNonExistentFilename();
     String alternativeFilename = generateNonExistentFilename();
 
-    List<String> result =
-        DynatraceMetadataEnricher.getMetadataFileContentWithRedirection(
-            filename, alternativeFilename);
-    assertEquals(Collections.<String>emptyList(), result);
+    Properties result =
+        DynatraceMetadataEnricher.getPropertiesWithIndirection(filename, alternativeFilename);
+    assertEquals(new Properties(), result);
```

Comment:
Isn't it the same as comparing to `isEmpty`, like above?

Reply:
True, changed it to `isEmpty` everywhere (5f577a0)